tests/e2e/utils/auth.setup.ts

```diff
@@ -0,0 +1,43 @@
+import { test as setup } from "@playwright/test";
+
+import { adminPassword, adminUser, statePassword, stateUser } from "./consts";
+
+const adminFile = "playwright/.auth/admin.json";
+
+setup("authenticate as admin", async ({ page }) => {
+  await page.goto("/");
+  const emailInput = page.getByRole("textbox", { name: "email" });
+  const passwordInput = page.getByRole("textbox", { name: "password" });
+  const loginButton = page.getByRole("button", { name: "Log In with Cognito" });
+  await emailInput.fill(adminUser);
+  await passwordInput.fill(adminPassword);
+  await loginButton.click();
+  await page.waitForURL("/");
+  await page
+    .getByRole("heading", {
+      name: "View State/Territory Reports",
+    })
+    .isVisible();
+  await page.waitForTimeout(1000);
```

Comment:
I would prefer to find something to wait for that would signal what we're waiting for is done, as set times are unreliable.

Reply:
I agree on principle! In the above 2 lines we are waiting for elements to be visible (and they would only be visible if logged in), but in this case it appears to not be enough...it's like the browser just needs a beat to completely finish saving the cookies before moving to the next test.